sessions/react-nesting/navigation/src/App.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "./styles.css";
+
+import avatar from "./img/avatar.jpg";
+import logo from "./img/logo.jpg";
+
+export default function App() {
+  return (
+    <header className="header">
+      <a href="#">
+        <img className="round-image" src={logo} alt="logo" />
+      </a>
+      <nav>
+        <a className="navigation__link" href="#home">
+          Home
+        </a>
+        <a className="navigation__link" href="#about">
+          About
+        </a>
+        <a className="navigation__link" href="#impressum">
+          Impressum
+        </a>
+      </nav>
+      <button
+        type="button"
+        onClick={() => console.log("I could toggle a profile!")}
+        aria-label="toggle profile"
+      >
+        <img className="round-image" src={avatar} alt="avatar image" />
+      </button>
+    </header>
+  );
```

Comment:
add some more context to the app, so that the component is not completely empty after extraction

```suggestion
  return (
    <>
      <header className="header">
        <a href="#">
          <img className="round-image" src={logo} alt="logo" />
        </a>
        <nav>
          <a className="navigation__link" href="#home">
            Home
          </a>
          <a className="navigation__link" href="#about">
            About
          </a>
          <a className="navigation__link" href="#impressum">
            Impressum
          </a>
        </nav>
        <button
          type="button"
          onClick={() => console.log("I could toggle a profile!")}
          aria-label="toggle profile"
        >
          <img className="round-image" src={avatar} alt="avatar" />
        </button>
      </header>
      <main>content goes here…</main>
    </>
  );
```

Reply:
this also removes "image" from the avatar alt text since eslint complains about that